build/common.sh

```diff
@@ -26,13 +26,13 @@ USER_ID=$(id -u)
 GROUP_ID=$(id -g)
 
 DOCKER_OPTS=${DOCKER_OPTS:-""}
-DOCKER=(docker ${DOCKER_OPTS})
+DOCKER=(docker "${DOCKER_OPTS}")
```

Comment:
I think this will break functionality. We want this var to be split

Reply:
```suggestion
IFS=" " read -r -a DOCKER <<< "docker ${DOCKER_OPTS}"
```

Reply:
thanks for the help